### PR TITLE
Update Bridge version + dev fixture data

### PR DESCRIPTION
**Pull request by mikiquantum**
This includes latest changes on bridge pallet:
- Wrapped admin operations into private functions
- Enable admin operations to use council voting

Allow genesis setup on dev/local/fulvous to whitelist bridge data.

### CODE REVIEW

File: runtime/src/bridge.rs [hunk 1]
```diff
@@ -20,6 +20,18 @@ pub trait Trait: system::Trait + chainbridge::Trait {
...
 
+decl_storage! {
+	trait Store for Module<T: Trait> as Bridge {}
+
+	add_extra_genesis {
+        config(chains): Vec<u8>;
+        config(relayers): Vec<T::AccountId>;
+        config(resources): Vec<ResourceId>;
+
+        build(|config| Module::<T>::initialize_test(&config.chains, &config.relayers, &config.resources))
```
Comment (philipstanislaus) [inline runtime/src/bridge.rs:31]: Let’s call this just `initialize`, since it can also be used outside of tests

File: runtime/src/lib.rs [hunk 2]
```diff
@@ -566,7 +566,8 @@ impl chainbridge::Trait for Runtime {
     type Event = Event;
     type Proposal = Call;
     type ChainId = ChainId;
-
+	/// A straight majority of the council can update bridge settings.
+	type AdminOrigin = pallet_collective::EnsureProportionAtLeast<_1, _2, AccountId, CouncilCollective>;
```
Comment (philipstanislaus) [inline runtime/src/lib.rs:570]: I would require >¾ here, for security purposes. The bridge holds a lot of value, better be safe than sorry ;)

File: runtime/src/nfts.rs [hunk 3]
```diff
@@ -158,10 +159,15 @@ mod tests {
...
 
+    ord_parameter_types! {
+		pub const One: u64 = 1;
+	}
+
 	impl chainbridge::Trait for Test {
 		type Event = ();
 		type Proposal = Call;
 		type ChainId = TestChainId;
+        type AdminOrigin = EnsureSignedBy<One, u64>;
```
Comment (philipstanislaus) [inline runtime/src/nfts.rs:170]: Indentation is somehow broken?

File: runtime/src/lib.rs (hunk 2 above)
Comment (mikiquantum) [reply runtime/src/lib.rs:570]: makes sense.

File: runtime/src/nfts.rs (hunk 3 above)
Comment (mikiquantum) [reply runtime/src/nfts.rs:170]: I think is the classic tale of tabs vs spaces. My IDE transforms tabs in spaces, but I believe the initial code was with tabs. It looks good on an editor anyways.

**Pull request merged**